osgbuild/mock.py

```diff
@@ -124,6 +126,23 @@ def _init_get_cfg_path(self):
                 os.path.join(cfg_dir,"mock-koji-%s-%s.%d.cfg" % (mock_config_from_koji, arch, os.getuid())),
                 mock_config_from_koji,
                 distro_tag)
+            # Can't really do this via the command line so hack up the mock config.
```

Comment:
```suggestion
            # Can't really do this via the command line (unless perl) so hack up the mock config.
```

Reply:
that should go without saying :-)

Reply:
Ah, you are right.  What happens in perl stays in perl.